tests/unittests/utilities/test_utilities.py

```diff
@@ -171,12 +171,11 @@ def test_recursive_allclose(inputs, expected):
 
 
 @pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU")
-@pytest.mark.xfail(sys.platform == "win32", reason="test will only fail on non-windows systems")
+@pytest.mark.xfail(sys.platform != "win32" and _TORCH_LESS_THAN_2_6, reason="test will only fail on non-windows systems")
 def test_cumsum_still_not_supported(use_deterministic_algorithms):
-    """Make sure that cumsum on gpu and deterministic mode still fails.
+    """Make sure that cumsum on GPU and deterministic mode still fails.
 
     If this test begins to pass, it means newer Pytorch versions support this and we can drop internal support.
```

Comment:
@SkafteNicki, I'm looking at the code of `_cumsum`. It already includes the implementation for case torch supports this natively, so there is no change needed, right?

Reply:
just for future reference, I repost slack discussion:

if I remember correctly the test is there to inform is if torch.cumsum suddenly gets native support for that specific case. In that case we can drop our own implementation. So it seems we now know that in the far future when I we only support Pytorch 2.6+ when can remove our own implementation